apps/leaguechamps/league_champs.star

```diff
@@ -0,0 +1,147 @@
+"""
+Applet: League Champs
+Summary: Display league characters
+Description: Shows league of legends champsions and their subtitle.
+Author: xl0lli
+"""
+
+load("cache.star", "cache")
+load("encoding/base64.star", "base64")
+load("encoding/csv.star", "csv")
+load("http.star", "http")
+load("random.star", "random")
+load("render.star", "render")
+load("schema.star", "schema")
+
+CACHE_TTL = 604800
+CSV_ENDPOINT = "https://gist.githubusercontent.com/xl0lli/bc6755ee77e52a9dcd481e75c95c74b6/raw/ac2c83cc874b2dbbe8f9449a8cef1df4ff8bedb4/league_champ_data"
+ANITMATION_SPEED = 200
+
+def main(config):
+    sprite_position = config.str("sprite_position", "random")
```

Comment:
```suggestion
    random.seed(time.now().unix // 15)

    sprite_position = config.str("sprite_position", "random")
```

Reply:
This would allow the app to be random, but deterministic over 15s time windows. This in turn allows our backend to cache the generated graphics more easily, which would be greatly appreciated!